geomesa-dynamo/geomesa-dynamodb-datastore/pom.xml

```diff
@@ -0,0 +1,164 @@
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>com.jcabi</groupId>
+                                    <artifactId>DynamoDBLocal</artifactId>
+                                    <version>2015-07-16</version>
+                                    <type>zip</type>
+                                    <outputDirectory>${project.build.directory}/dynamodb-dist</outputDirectory>
+                                    <overWrite>false</overWrite>
+                                </artifactItem>
+                            </artifactItems>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>1.10</version>
+                <executions>
+                    <execution>
+                        <id>reserver-dynamodb-port</id>
+                        <goals>
+                            <goal>reserve-network-port</goal>
+                        </goals>
+                        <configuration>
+                            <portNames>
+                                <portName>dynamodblocal.port</portName>
+                            </portNames>
+                            <randomPort>true</randomPort>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.jcabi</groupId>
+                <artifactId>jcabi-dynamodb-maven-plugin</artifactId>
+                <version>0.7.2</version>
+                <executions>
+                    <execution>
+                        <id>dynamodb-integration-test</id>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <port>${dynamodblocal.port}</port>
+                            <dist>${project.build.directory}/dynamodb-dist</dist>
+                            <arguments>
+                                <argument>-inMemory</argument>
+                                <argument>-sharedDb</argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>2.19.1</version>
+                <configuration>
+                    <includes>
+                        <include>**/*IT.java</include>
+                        <include>**/*IT.scala</include>
+                        <include>**/*IT.class</include>
```

Comment:
seems like you would want to include either source files or class files, not both...